generators/generator-base.js

```diff
@@ -27,6 +27,8 @@ const exec = require('child_process').exec;
 const os = require('os');
 const pluralize = require('pluralize');
 const jhiCore = require('jhipster-core');
+const dateFormatForLiquibase = require('jhipster-core/lib/utils/format_utils').dateFormatForLiquibase;
```

Comment:
this isn't to be imported you know :) I believe there's a piece of code somewhere in this repo that does this

Reply:
Removed the commit.
I was removing a duplicated code between jhipster-core and generator-jhipster.

Reply:
Don't worry, all of this will be refactored very soon when/if JCore and the generator are merged, but thanks!!